graphql/schema.graphql

```diff
@@ -13,7 +13,7 @@ type Query {
   taskAllExecutions(taskId: String!): [Task!]!
   patch(id: String!): Patch!
   version(id: String!): Version!
-  projects: [GroupedProjects]! 
+  projects: [GroupedProjects]!  @superUserOnly
```

Comment:
I'm not sure if you did this for testing / what this is used for but I wouldn't think this is necessarily superUser only.

Reply:
nice catch